sdk/core/System.ClientModel/src/Options/ClientLoggingOptions.cs

```diff
@@ -55,10 +54,9 @@ public class ClientLoggingOptions
             "Transfer-Encoding",
             "User-Agent",
             "WWW-Authenticate" };
-
-    internal static string[] DefaultAllowedQueryParameters { get; } = new[] { "api-version" };
-
+    private static string[] DefaultAllowedQueryParameters { get; } = new[] { "api-version" };
     internal static PipelineMessageSanitizer DefaultSanitizer { get; } = new(DefaultAllowedHeaderNames, DefaultAllowedQueryParameters);
+    internal const double RequestTooLongSeconds = 3.0; // sec
```

Comment:
I just moved this to make it easier for me to identify groupings of fields in the type ... no significant change here.